### PR TITLE
Speakers page sessions now redirects to particular session

### LINKED ISSUE #1467: Fix sessions in speakers field for single page config
**Actual Behaviour**
When the user clicks on the a session from speakers page it takes the user to tracks page in case of single session page events

**Expected Behaviour**
The sessions on speakers page should open in separate session page like in schedule page.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1467 #1482

Changes: Changed redirect url for sessions on speakers page and wrote selenium test for jumpToSession.

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@Princu7 @geekyd please review.

**Comment by agbilotia1998** [comment]
@Princu7 @geekyd please guide on this , as I think there is no test written for jumping to session page.If it is so then there remains two options-
1. Removing this test from speaker.js
2. Writing jumpToSession() test.

**Comment by geekyd** [comment]
@agbilotia1998 We should add test instead of removing the existing tests @Princu7 what do you think about this?

**Comment by agbilotia1998** [comment]
@geekyd @Princu7 please review.All 75 tests are passing but still travis is failing please check:)

**Comment by geekyd** [comment]
Looks good to me the tests are failing due to an issue in travis. @agbilotia1998 Can you try the work-around provided here https://github.com/travis-ci/travis-ci/issues/8082#issuecomment-315147561

**Comment by agbilotia1998** [comment]
@geekyd I tried travis work-around but it didn't work.
PS: https://github.com/fossasia/open-event-webapp/pull/1474/commits/b3640280604dade87122d9c68332a615d0a53f63

**Comment by geekyd** [comment]
@agbilotia1998 Please fix the codacy issues https://www.codacy.com/app/mb/open-event-webapp/pullRequest?prid=950452

**Comment by agbilotia1998** [comment]
@geekyd done.Please review.

File: .travis.yml [hunk 1]
```diff
@@ -21,12 +21,11 @@ addons:
 install:
   - npm install -g istanbul mocha
   - npm install
-script:
```
Comment (geekyd) [inline .travis.yml]: @agbilotia1998  Why are we removing this?

**Comment by agbilotia1998** [comment]
@geekyd actually I was testing what is making travis fail , that is why did that , I have restored it from development branch please check.

**Comment by mariobehling** [comment]
Squashing the commits for you. Please do so next time. Cheers. Keep it up!

**Pull request merged**